rs-backend/src/api.rs

```diff
@@ -0,0 +1,167 @@
+// Types implement both serialize and deserialize despite conceptually needing
+// only one for testing.
+
+pub mod summary {
+    use std::collections::HashMap;
+
+    /// One decimal place rounded percent
+    type Percent = String;
```

Comment:
Hmm, yeah, of course. Still, I think it would be nice to implement deserialise so that we can verify that it fits our conception of a percentage - using a String seems kinda risky.